tests/runner.py

```diff
@@ -156,11 +156,16 @@ def decorated(self, *args, **kwargs):
   return decorated
 
 
+# TODO(sbc): Investigate all tests with this decorator and either fix of remove the test.
 def no_wasm_backend(note=''):
   assert not callable(note)
 
   def decorated(f):
-    return skip_if(f, 'is_wasm_backend', note)
+    @wraps(f)
+    def skip(self, *args, **kwargs):
+      return self.skipTest(note)
```

Comment:
this is pretty confusing - `def decorated`, `@wraps`, and also `def skip` - are all these layers needed? I think the other ones are simpler? (e.g. `is_slow_test` right above it has one fewer `def`)

Reply:
I found waaay simpler way to do it.